Module 4

lk4n opened the pull request.

lk4n committed:
Not ready but started

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

lk4n committed:
Fixed few tests
Add update()
Fixed loading state
Fixed remove()
Add destroy()

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

lk4n committed:
Update remove, destroy

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

lk4n committed:
Add charts getter

lk4n committed:
Update charts() to be more readable

lk4n committed:
Method updates

Update getTemplate() to getter - get template()
Renamed charts() to chartColumns()

lk4n committed:
Add subElement

jsru commented:
Добавляю преподавателя (@dosandk) для код-ревью.

lk4n committed:
Add anchor() method for links

jsru commented:
Решение было обновлено, посмотрим что скажет @dosandk

dosandk commented on 04-oop-basic-intro-to-dom/1-column-chart/index.js:
```diff
@@ -1,3 +1,104 @@
+  }
+
+  get anchor() {
+    let result = "";
```
может лучше использовать тернарник?

dosandk reviewed (approved):
👍 good
